### ISSUE TITLE
When in incognito, Move Tabs to Other Window doesn't work.

### ISSUE THREAD

**Issue by celphtilted** [piroor/multipletab#210]
So I don't know if its Firefox itself that does not allow this, but if I try to do it on a tab from incognito and transfer it to a non-incognito window, it opens a blank new tab on the non-incognito window, but it doesn't move the tab. If I try the other option to transfer it to a completely new window, the tab transfers seemlessly, however it is transfered to another incognito window not a normal one. A note tho from my testing, if I first try to move it to an existing Incognito to Normal Window, and only then try to move the tab to a completetly new (incognito) window, it won't work anymore, nothing happens, also the tab cannot be left or right clicked anymore, the blank tab doesn't even appear, I guess it is already flagged as a moved tab at that point, but again, it is not. All other tabs that were not touched work normally and are interactive.

For me personally, let's say my most common example would be I sometimes use Incognito to log in to some account without logging out from another, mostly when a mate uses my PC but I often do that too, now while in that session I start a chain of tabs, sometimes intentionally, sometimes by mistake in that same incognito window, so now what happens is if I get a random crash, power cut, I lose all my content, or if I quickly wanna turn off the PC or restart it due to an update, I need to transfer all the tabs manually from the Incognito window to my normal window, and sometimes this takes so much time especially if multiple instances of incognito tab chains have been open for a few days, again some that I forgot, some with intent of cookies declutter and whatnot. So in that scenario, this is something that will be very useful to me so if its not Firefox fault that this happens, besides the two options that we have now, Move Tabs to New Window & Move Tabs to Other Window, I would like to see an option called something like Move Tabs to Incognito Window, and also make possible for incognito tabs to be transferred to other non-incognito windows with the the context menu option that we already have now have, again if its possible within Firefox.

Props, I appreciate your work.

**Comment by piroor** [comment]
Basically Firefox cannot move an existing tab between different type windows: private browsing and normal browsing. You'll see the restriction by dragging a tab on the tab bar to another window across their mode - the drop target window won't accept the tab to be dropped.

Thus what I should do is: don't show different type windows in the list of "Move to Other WIndows".

**Comment by piroor** [comment]
> Thus what I should do is: don't show different type windows in the list of "Move to Other WIndows".

The commit 1ff9bd1 should do that.

**Comment by celphtilted** [comment]
Yep, I notice that I cannot drag them from one to another when in different mode hence I said it might be Firefox own issue. 

Now knowing that this is for sure restricted, is it possible for you to add a functionally that basically moves the URLs not the actual tabs when the mode differs (it can be noted that the URLs and not the tabs are moved)? Even tho its not the actual tab, most of the time the backlog that in my personal use case happens in private tab is something I want to be in normal browser, even better in some cases.
